src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
please write here modifiers like `calendar--month-length-29`  `calendar--start-sun`

maybe this names are not correct, please check in your scss how you called them

when you write this classes your calendar shoud changes

please watch this video:
https://www.loom.com/share/d0e6db0cd54441f6b8366d10121d03d1